components/style/properties/gecko.mako.rs

```diff
@@ -1490,8 +1490,10 @@ fn static_assert() {
     /// This function will also handle scriptminsize and scriptlevel
     /// so should not be called when you just want the font sizes to be copied.
     /// Hence the different name.
+    ///
+    /// Returns true if the inherited size was actually used
```

Comment:
Should probably clarify that you mean the inherited keyword-based size here (and not say the sizes in `parent`).